microprofile/lra/jax-rs/src/test/java/io/helidon/microprofile/lra/LoadBalancedCoordinatorTest.java

```diff
@@ -95,6 +96,7 @@
 // LRA client
 @AddExtension(LraCdiExtension.class)
 // Test resources
+@AddBean(CatchAllExceptionMapper.class)
```

Comment:
CatchAllExceptionMapper was removed from Helidon, and test requires it.

Reply:
See comment above